R/utils-json.R

```diff
@@ -263,10 +263,15 @@ subsetByMonth <- function(pts, onlyMonth) {
   }
   return(pts)
 }
+
 getYvals_approvals <- function(object, num_vals){
   ylim <- ylim(object)$side.2[1]
   yvals <- rep(ylim, num_vals)
-  return(yvals)
+  
+  # The constant scaling factor below was reverse-engineered from analyzing the 
+  # (pandoc?) SVG in Inkscape. I'm not entirely sure why it works, and it's easy
+  # to imagine why it wouldn't in some cases.
+  return(0.908196 * yvals)
```

Comment:
until we can get a helper-type function the just retrieves the user extent for us